src/Controls/StreetControls.js

```diff
@@ -63,6 +57,14 @@ class StreetControls extends FirstPersonControls {
         this.keyGoToNextPosition = 90;
         this.keyGoToPreviousPosition = 83;
         this.keySetCameraToCurrentPosition = 65;
+
+        // Raycaster used to pick objects
+        this.raycaster = new THREE.Raycaster();
+        // Tween is used to make smooth animations
+        this.tweenGroup = new TWEEN.Group();
```

Comment:
Declare `this.tweens` (which should be `this.tween`) here. Side note: why do you need a group and a tween ? (I know nothing about this library)

Reply:
ok. side answer: First I try without a Group but I didn't manage to make it work.